flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/ResultPartitionFactoryTest.java

```diff
@@ -39,23 +40,22 @@
 public class ResultPartitionFactoryTest extends TestLogger {
 
 	@Test
-	public void testForceConsumptionOnReleaseEnabled() {
-		testForceConsumptionOnRelease(true);
+	public void testConsumptionOnReleaseEnabled() {
+		testConsumptionOnRelease(true);
```

Comment:
if `testConsumptionOnRelease` returned `ResultPartition`, you could do the assertion here.
Also `testConsumptionOnRelease ` could directly accept the `ReleaseType` then there would be less ifs.